### ISSUE TITLE
Source Orbit: Failed to parse file with TypeError: Cannot read properties of undefined (reading 'library')

### ISSUE THREAD

**Issue by AG1965** [codefori/vscode-rpgle#287]
called "so" in a directory with several subdirectories with our sources inside when this happended:
```
qclpsrc/#chkv3b.pgm.clle - Failed to parse file.
qclpsrc/#chkv3b.pgm.clle
TypeError: Cannot read properties of undefined (reading 'library')
    at /QOpenSys/pkgs/lib/nodejs20/lib/node_modules/@ibm/sourceorbit/dist/index.js:3:87890
    at Array.forEach (<anonymous>)
    at t.Targets.createClTarget (/QOpenSys/pkgs/lib/nodejs20/lib/node_modules/@ibm/sourceorbit/dist/index.js:3:87857)
    at t.Targets.parseFile (/QOpenSys/pkgs/lib/nodejs20/lib/node_modules/@ibm/sourceorbit/dist/index.js:3:83515)
    at async /QOpenSys/pkgs/lib/nodejs20/lib/node_modules/@ibm/sourceorbit/dist/index.js:3:76285
[ERROR] Report this issue to us with an example: github.com/halcyon-tech/vscode-rpgle/issues
```
copied the problematic file #chkv3b.pgm.clle to a new empty directory and called "so" in there, the error is repeatable. 
(The source can be successfully compiled with CRTBNDCL.)
This is the file, added ending .txt to please github file upload.
[#chkv3b.pgm.clle.txt](https://github.com/codefori/vscode-rpgle/files/13931145/chkv3b.pgm.clle.txt)

**Comment by worksofliam** [comment]
Thanks for your issue! I will look into this on Tuesday. Appreciate it!

**Comment by worksofliam** [comment]
Hey @AG1965 !!

This is fixed in SO 0.6.1 now: https://github.com/IBM/sourceorbit/releases/tag/0.6.1

Please raise more issues over on that repo :)

Thanks!!
